psalm-baseline.xml

```diff
@@ -25,6 +17,24 @@
             <code>array_merge(array_flip($keys), $this-&gt;elements)</code>
         </InvalidPropertyAssignmentValue>
     </file>
+    <!-- https://github.com/vimeo/psalm/issues/4963 -->
+    <file src="src/Show/ShowMapper.php">
+        <InvalidParamDefault occurrences="1">
+            <code>FieldDescriptionOptions</code>
+        </InvalidParamDefault>
+    </file>
+    <!-- Will be fixed in Symfony 5.3 https://github.com/symfony/symfony/pull/40604 -->
+    <file src="src/Translator/Extractor/AdminExtractor.php">
+        <MoreSpecificImplementedParamType occurrences="1">
+            <code>$resource</code>
+        </MoreSpecificImplementedParamType>
+    </file>
+    <!-- BC for Symfony 4 since InputBag was introduce in Symfony 5 -->
+    <file src="tests/Admin/AdminHelperTest.php">
+        <PropertyTypeCoercion occurrences="4">
+            <code>new ParameterBag();</code>
+        </PropertyTypeCoercion>
+    </file>
```

Comment:
In this case,

https://github.com/sonata-project/SonataAdminBundle/blob/master/tests/Admin/AdminHelperTest.php#L175-L189

I think it would be better to create a real request and there is no need to call `new ParameterBag()`.

Reply:
Indeed, it should be ok now